### PR TITLE
Flyout review and changes

### PR DESCRIPTION
**Pull request by tashatitova**
- Moving static resources from theme to static
- Removing extra resources
- Updating HC resources to correct brushes and stroke width (2 in HC)

Padding (of default 0) seemed to have been set twice, so actually moving it to template binding padding on outer border.

This is an empty container flyout - a lightweight control with no additional content.

### CODE REVIEW

File: dev/CommonStyles/FlyoutPresenter_themeresources.xaml [hunk 1]
```diff
@@ -7,29 +7,24 @@
...
-            <Thickness x:Key="FlyoutBorderThemePadding">0</Thickness>
         </ResourceDictionary>
         <ResourceDictionary x:Key="HighContrast">
-            <StaticResource x:Key="FlyoutPresenterBackground" ResourceKey="SystemControlPageBackgroundChromeMediumLowBrush" />
-            <StaticResource x:Key="FlyoutBorderThemeBrush" ResourceKey="SystemControlTransientBorderBrush" />
-            <Thickness x:Key="FlyoutBorderThemeThickness">1</Thickness>
-            <Thickness x:Key="FlyoutBorderThemePadding">0</Thickness>
+            <StaticResource x:Key="FlyoutPresenterBackground" ResourceKey="SystemColorWindowColorBrush" />
+            <StaticResource x:Key="FlyoutBorderThemeBrush" ResourceKey="SystemColorWindowTextColorBrush" />
+            <Thickness x:Key="FlyoutBorderThemeThickness">2</Thickness>
```
Comment (karenbtlai) [inline dev/CommonStyles/FlyoutPresenter_themeresources.xaml:17]: Wondering why the changes to HC brushes? Did these come from design?

File: dev/CommonStyles/FlyoutPresenter_themeresources.xaml [hunk 2]
```diff
@@ -54,13 +49,12 @@
             <Setter.Value>
                 <ControlTemplate TargetType="FlyoutPresenter">
                     <Border
-                        Background="{TemplateBinding Background}"
-                        contract7Present:BackgroundSizing="OuterBorderEdge"
+                        Background="{TemplateBinding Background}"                        
                         BorderBrush="{TemplateBinding BorderBrush}"
                         BorderThickness="{TemplateBinding BorderThickness}"
                         contract7Present:CornerRadius="{TemplateBinding CornerRadius}"
-                        contract7NotPresent:CornerRadius="{ThemeResource OverlayCornerRadius}"
```
Comment (karenbtlai) [inline dev/CommonStyles/FlyoutPresenter_themeresources.xaml:62]: Wondering why this is removed?

Comment (tashatitova) [reply dev/CommonStyles/FlyoutPresenter_themeresources.xaml:62]: It's a bug. We don't have corner radius before win7

File: dev/CommonStyles/FlyoutPresenter_themeresources.xaml (hunk 1 above)
Comment (tashatitova) [reply dev/CommonStyles/FlyoutPresenter_themeresources.xaml:17]: We don't want to use extra levels of abstractions through old SystemControl brushes. Instead, use directly proper HC where we manually touch the code.

Comment (marksfoster) [inline dev/CommonStyles/FlyoutPresenter_themeresources.xaml:17]: Wondering why the border thickness for HC has changed from 1 to 2

Comment (mdtauk) [reply dev/CommonStyles/FlyoutPresenter_themeresources.xaml:17]: High contrast benefits from a bolder outline I would guess.

Comment (tashatitova) [reply dev/CommonStyles/FlyoutPresenter_themeresources.xaml:17]: All HC borders should be 2px wide per accessibility guidance, unless it's some sort of a custom/smaller element

File: dev/CommonStyles/FlyoutPresenter_themeresources.xaml [hunk 3]
```diff
@@ -54,13 +49,12 @@
...
-                        Padding="{ThemeResource FlyoutBorderThemePadding}">
+                        contract7Present:BackgroundSizing="OuterBorderEdge"
+                        Padding="{TemplateBinding Padding}">
```
Comment (marksfoster) [inline dev/CommonStyles/FlyoutPresenter_themeresources.xaml:57]: `Padding="{TemplateBinding Padding}"` will push the scroll viewer in away from the edge of the flyout which I don't think we want.

**Review by marksfoster — APPROVED** [review]

File: dev/CommonStyles/FlyoutPresenter_themeresources.xaml (hunk 3 above)
Comment (mdtauk) [reply dev/CommonStyles/FlyoutPresenter_themeresources.xaml]: The new design for the scroll viewer has rounded corners, so it may be intended

Comment (tashatitova) [reply dev/CommonStyles/FlyoutPresenter_themeresources.xaml]: I've updated the way Mark suggested. We want scrollviewer to be flush, but then the content has padding (which a dev can customize via template).
